cli/node_identity.go

```diff
@@ -0,0 +1,46 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package cli
+
+import (
+	"github.com/spf13/cobra"
+)
+
+func MakeNodeIdentityCommand() *cobra.Command {
+	var cmd = &cobra.Command{
+		Use:   "node_identity",
+		Short: "Get information about the node's identity",
+		Long: `Get information about the node's identity.
```

Comment:
suggestion: I think it is worth noting in the `Short` and first line of `Long` that this command only returns the public components of the node's identity.  Unless perhaps you have immediate plans to also yield the private key too in a follow-up PR.

Maybe just change it to:
> Get the public information about the node's identity

Otherwise this info is buried (and even then only by omission) within the body of `Long`.

Reply:
> plans to also yield the private key too in a follow-up PR.

We should not be dumping priv key IMO, I raised that previously on this PR as well.